config/eye.yml.erb

```diff
@@ -79,4 +90,4 @@ processes:
         config:
           times: 3
           every: 20 seconds
-          below: 1 gigabyte # Change this per installation based on box size
```

Comment:
Curious why you lowered this? On the other installs it is a gig

Reply:
This instance gets almost no traffic, since it's not used for active development and testing.  Also I wasn't confident about the available memory on `tablexi-shared02`.